src/main/java/koval/myfit/mobile/gui/pages/service/AdbService.java

```diff
@@ -0,0 +1,24 @@
+package koval.myfit.mobile.gui.pages.service;
+
+import java.io.IOException;
+
+
+import static koval.myfit.mobile.gui.pages.service.interfaces.IConstantUtils.ADB_PATH;
+
+public class AdbService {
+
```

Comment:
Create here enum AppPackage and try to implement

  private static AndroidService executor = AndroidService.getInstance();

    public void clearAppCache(AppPackage packageName) {
        LOGGER.info("Clear ['{}'] cache via ADB", packageName);
        String[] command = { "adb", "-s", getDevice().getUdid(), "shell", "pm", "clear", packageName.getPackageName() };
        getDevice().execute(command);
    }
    
        public void startApp(AppPackage packName) {
        LOGGER.info("Starting " + packName.toString());
        String cmd = String.format("shell monkey -p %s -c android.intent.category.LAUNCHER 1", packName.getPackageName());
        executor.executeAdbCommand(cmd);
        pause(TimeConstants.FIVE_SEC_TIMEOUT);
    }

Reply:
I had trouble with this task, but I solved it! 
1. Error [Cannot run program "adb": error=2, No such file or directory] -> write ANDROID_HOME path to .zshrc file worked for me
2. getDevice() return null -> getDevice(getDriver)